### ISSUE TITLE
paper-select: content is not re-positioned when options changes

### ISSUE THREAD

**Issue by panthony** [miguelcobain/ember-paper#939]
Using the select's search function, the displayed items can changes.

When it happens the select's content is not re-positioned to take into account its new size.

**Comment by panthony** [comment]
This is strictly the same issue as #923

This is fixed (not sure if broke anything) by adding this in `calculatePosition`:

```js
function resetPosition(containerNode) {
  containerNode.style.top = '';
  containerNode.style.left = '';
  containerNode.style.transform = '';
  containerNode.style.transformOrigin = '';
}

  calculatePosition(trigger, content, destination, { dropdown }) {
    // only when repositioning 
    if (!dropdown.didAnimateScale) {
      resetPosition(content);
    }
    // ...
}
```

This is kind of hard to unit test, not sure if you are interested in a PR or not

**Comment by patodevilla** [comment]
I'm still having a lot of issues with paper-menu. It seems that the smallest change in window size will cause it's position to go off (this happens also in the demo page). I'll try to find out what is causing this issue as soon as I have some time. 

Meanwhile, I completely commented the calculatePosition function and simply changed some css.

md-menu-content, .ember-basic-dropdown-content{
  width: 300px;
}
.md-open-menu-container{
  left: auto;
}

Also, I noticed that the component can be improved when using Ember Concurrency tasks, since the menu-item component gets destroyed onClick and I found no unintrusive way to avoid this.
